### PR TITLE
Migration to k4geo

### PR DESCRIPTION
**Pull request by giovannimarchiori**
Code changes related to k4geo migration of theta-module segmentation

### CODE REVIEW

File: FCCSW_ecal/neighbours_theta.py [hunk 1]
```diff
@@ -4,12 +4,12 @@
 # Detector geometry
 from Configurables import GeoSvc
 geoservice = GeoSvc("GeoSvc")
-# if FCC_DETECTORS is empty, this should use relative path to working directory
-path_to_detector = os.environ.get("FCCDETECTORS", "")
+# if K4GEO, this should use relative path to working directory
```
Comment (BrieucF) [inline FCCSW_ecal/neighbours_theta.py:7]:
```suggestion
# if K4GEO is empty, this should use relative path to working directory
```

**Review by BrieucF — COMMENTED** [review]
Thanks! After adding the rootfiles and updating the recipe this should be good to go!

**Comment by BrieucF** [comment]
Can you also add the two files equivalent to these one: https://github.com/BrieucF/LAr_scripts/tree/main#getting-lar_scripts-package-and-the-required-data-files and update the README ?

**Comment by giovannimarchiori** [comment]
I added the small file in data/ and the big files in Eos and updated the README

File: .gitignore [hunk 2]
```diff
@@ -139,5 +139,5 @@ dmypy.json
 
 # my stuff 
 *.sub
-*.root
+
```
Comment (giovannimarchiori) [inline .gitignore:142]: I removed the *.root line because it's duplicate (there is the same line at the top of the file)

**Comment by giovannimarchiori** [comment]
Hi @BrieucF - just to be sure, you are satisfied by these changes and the PR is ready to be merged once https://github.com/key4hep/k4geo/pull/304 and https://github.com/HEP-FCC/k4RecCalorimeter/pull/59 are merged?

Thanks, Giovanni

**Comment by BrieucF** [comment]
Hi Giovanni, yes! And I'll do a final check once the other PR are merged.

**Comment by BrieucF** [comment]
Thanks Giovanni!

**Pull request merged**